src/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render } from 'react-dom'
+
+class App extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      givenTemp: 82,
+      convertToCelsius: true
```

Comment:
Using a boolean makes the assumption you will always convert between Fahrenheit and Celsius. #KelvinNeedsLoveToo